ipn/ipnserver/server.go

```diff
@@ -73,6 +73,14 @@ type Options struct {
 	// ErrorMessage, if not empty, signals that the server will exist
 	// only to relay the provided critical error message to the user.
 	ErrorMessage string
+
+	// GetEngine optionally provides a "try again" hook for the
```

Comment:
Any chance we could make this non-optional? Optional means there's one more branch in this already complex state machine. It'd be nice to avoid that.

Reply:
Done.